File: C#/FrameworksPerformance/Common/Facades/IRelationalFrameworkTestsFacade.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common.Facades
{
    public interface IRelationalFrameworkTestsFacade : IFrameworkTestsFacade
    {
        string SearchTwoRelatedTables(int samplesQuantity);
        string SearchFourRelatedTables(int samplesQuantity);
        string SearchRecordsWhichDoesNotHaveConnection(int samplesQuantity);
        string SearchWithSubQuery(int samplesQuantity);
    }
}

File: Java-sqlite/JavaDatabaseFrameworks/Common/src/main/java/org/dbFrameworks/Facade/IRelationalFrameworkTestsFacade.java
package org.dbFrameworks.Facade;

public interface IRelationalFrameworkTestsFacade extends IFrameworkTestsFacade {
    String SearchTwoRelatedTables(int samplesQuantity);
    String SearchFourRelatedTables(int samplesQuantity);
    String SearchRecordsWhichDoesNotHaveConnection(int samplesQuantity);
    String SearchWithSubQuery(int samplesQuantity);
    String RemoveRelatedRecords(int samplesQuantity);
}
